### ISSUE TITLE
Developer feedback request > Updates to Login.gov developer documentation

### ISSUE THREAD

**Issue by indexing** [cal-itp/benefits#1676]
Login.gov is seeking developer feedback on the latest updates to the OIDC guide. I've attached the draft they shared.
[External Review_OIDC Full Draft_Read-only.pdf](https://github.com/cal-itp/benefits/files/12381414/External.Review_OIDC.Full.Draft_Read-only.pdf)

Capacity permitting, it would be great if we can share our insights.

## Acceptance Criteria

- [x] [Feedback questionnaire](https://forms.gle/6cLpSh7k3A2S2crZ9) is complete and submitted.

## Additional context

@vykster brought this [post](https://cal-itp.slack.com/archives/C037Y3UE71P/p1692310639015019) to our attention in the Cal-ITP #benefits-identity channel.

**Comment by johnatstate** [comment]
I submitted the following in the form:
I did have one issue when first setting this up: the section on creating and uploading an openssl cert. I did as instructed, then when it came time for production, I was told that it wasn't needed since we are using PKCE, which made sense. I use PKCE for all the other OIDC test apps and have no certs uploaded. I noticed the revised instructions still indicate that creating and uploading a cert is a required step but doesn't mention under which conditions, i.e. private_key_jwt.
